Add initial goals, strategy, and structural documentation (#17)
jupyter-governance/jupyter-foundation-governing-board: doc/strategy.md

# Goals and strategy

> Our goal is to follow these strategic goals and priorities **through March of 2026**, with the plan of re-visiting this structure once we’ve gotten more experience and grown as a team.

Reviewer: ```suggestion
Our goal is to follow these strategic goals and priorities **through March of 2026**, with the plan of revisiting this structure once we’ve had more experience and grown as a team.
```

> ## Our 12 month goals

Reviewer: ```suggestion
## Our 12-month goals
```

> The Foundation began operations this year, and so we define the following 12-month goals:

Reviewer: ```suggestion
The Foundation ramped up its operation in early 2025. We define the following 12-month goals through early 2026:
```

> 1. Define the minimal structures needed to efficiently operate  

Reviewer: ```suggestion
1. Define the minimal structures needed to efficiently operate.
```

> 2. Make quick progress in using Foundation funds to support Jupyter’s community  

Reviewer: ```suggestion
2. Make quick progress in using Foundation funds to support Project Jupyter and its mission.  
```

3. Learn as quickly as we can.  

> 4. Prepare ourselves for more complex operations and decision making in the future.

Reviewer: ```suggestion
4. Prepare ourselves for more complex operations and decision-making in the future.
```

## Our strategy for achieving these goals

Our current strategy is a **short-term strategy**. It is focused on helping us take our first steps, to have initial impact with our funds, and to learn. Here are major strategic priorities:

1. **Focus on learning and taking quick, safe-to-try actions, rather than getting it perfect.** We’re a young team with a lot to learn, and our first job is to learn quickly how we can be most-effective in accomplishing our goals. It’s OK to make mistakes, as long as we learn, and as long as we are doing our best to live up to our goals.  

> 2. **Prioritize using funds to deliver value to the Jupyter community.** Prioritize disbursing funds in a way that brings more resources, skills, etc into the Jupyter community. If the Jupyter Foundation needs staff for its own operations, aim to prioritize these for year two so that we can focus on learning for now.  

Reviewer: ```suggestion
2. **Prioritize using funds to deliver value to Project Jupyter.** Prioritize disbursing funds in a way that brings more resources, skills, etc into the Jupyter community. If the Jupyter Foundation needs staff for its own operations, aim to prioritize these for year two so that we can focus on learning for now.  
```

3. **Identify opportunities that have a leveraged impact.** Use funding in ways that create force-multiplier effects. For example, by providing capacity that cuts across the sub-projects, by funding efforts that grow the overall capacity within Jupyter, etc. Don’t fund people just to continue an anti-pattern, but instead fund capacity to help Jupyter teams get *out* of anti-patterns.
4. **Start by building a strong foundation of team and technology practices.** Rather than starting with new features and functionality, begin by ensuring that Jupyter’s subprojects have the resources and guidance they need to follow transparent and effective software development practices, leading to high-quality software and teams.  

> 5. **Focus on listening and communication with both the Jupyter community and the member organizations, and provide a rationale for our decisions.** The Board should ensure that we actively communicate with the Jupyter community, and with the Foundation members. This will give us more community trust that makes it safer to spend funds in a creative and quick way.

Reviewer: ```suggestion
5. **Focus on listening and communication with both the Jupyter community and the member organizations, and provide a rationale for our decisions.** The Board should ensure that we actively communicate with the Jupyter community and with the Foundation members. This will give us more community trust, which will make it safer to spend funds in a creative and quick way.
```
